### ISSUE TITLE
paypal pro supported?

### ISSUE THREAD

**Issue by shaibal13** [thephpleague/omnipay#616]
is this version has paypal pro?

**Comment by carlos-mg89** [comment]
Yes, this is the package you are looking for https://github.com/thephpleague/omnipay-paypal

You can see in https://github.com/thephpleague/omnipay-paypal/tree/master/src that there are several Paypal gateways. And One of these ones is precisely for Paypal Pro.
